### DISCUSSION TITLE
小狼毫在千牛工作台的某些特殊场景无法输入文字

### DISCUSSION THREAD

**Discussion by thofd** [rime/home#1694] [General]
操作系统信息

OS 详细版本: Microsoft Windows [版本 10.0.19045.4598]
小狼毫版本: [ 如 0.16.3 ， CI 构建版请附短 commit hash ，如 372c9c ]
描述遇到的问题
简短地描述你所遇到的问题

复现步骤
重现问题的步骤，如：

打开 '千牛工作台' 软件，进入评价管理页面，打开“评价回复”
点击“新增加回复模板”
在新开的输入框中输入任意文字，输入框内都无法输入
预期行为
小狼毫文字选择框会闪一下消失，跟随键盘输入重复出现
![image](https://github.com/user-attachments/assets/f95d8e44-9c9f-44d2-8625-ca15c0aa7f60)

**Comment by ATang007ZH** [comment]
0.17可以了。但是候选框不能一直停留，只能展示1-2秒，之后就消失了。
输入要快一点才行。
